src/client/client4.ts

```diff
@@ -1449,7 +1449,14 @@ export default class Client4 {
         );
     };
 
-    getMyChannels = async (teamId: string) => {
+    getMyChannels = async (teamId: string, includeDeleted = false) => {
+        const serverVersion = this.getServerVersion();
+        if (isMinimumServerVersion(serverVersion, 5, 21)) {
+            return this.doFetch(
+                `${this.getUserRoute('me')}/teams/${teamId}/channels${buildQueryString({include_deleted: includeDeleted})}`,
```

Comment:
you can keep this without needed to check for the server version as adding a query string should have no  effect  on previous servers, or am I wrong?

Reply:
Yes, I've tested and no problem sending an unused query parameter. I was playing safe here